app/pages/CannotRemoveExistingPreviousRegistrationsPage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import models.UserAnswers
+import play.api.mvc.Call
+
+object CannotRemoveExistingPreviousRegistrationsPage extends Page {
+
+  override def route(waypoints: Waypoints): Call =
+    controllers.previousRegistrations.routes.CannotRemoveExistingPreviousRegistrationsController.onPageLoad()
+
+  override def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page =
+    throw new NotImplementedError("nextPageNormalMode is not implemented")
```

Comment:
Do we need this instead of keeping the default nextPageNormalMode implementation?